src/firefly/js/templates/lightcurve/LcManager.js

```diff
@@ -267,28 +272,37 @@ export function* lcManager(params={}) {
     }
 }
 
-function updateRawTableChart(timeCName, fluxCName) {
+
+function updateRawTableChart(timeCName, fluxCName, mission='') {
     var chartX = get(getChartDataElement(LC.RAW_TABLE), ['options', 'x', 'columnOrExpr']);
     var chartY = get(getChartDataElement(LC.RAW_TABLE), ['options', 'y', 'columnOrExpr']);
 
     if (chartX === timeCName && chartY === fluxCName) return;
 
     if (timeCName && fluxCName) {
-        const xyPlotParams = {x: {columnOrExpr: timeCName}, y: {columnOrExpr: fluxCName, options: 'grid,flip'}};
+
+        const  title = (mission && (mission.toLowerCase()==='wise' || mission.toLowerCase()==='other') )? 'Input Data':'';
```

Comment:
Agree. This need to change. We discussed yesterday the topic. Lijun will work on that refactoring.

Let's be clear(er) here, the agreement from the CCB was to change what the ticket explains to all the cases in LC. Yesterday I talked to Lijun so she can _make the common code not mission specific_, and only keep what is specific to mission **separately** in the already existing files which are used to control images and layout specific mission control panel. There shouldn't be any specific code in `LcManager.jsx`, `LcResult.jsx` and `LcViewer.jsx`, that was the original design/ideas. 
But, Lijun needs to add specific code that will deal with the change request so she can handle the specific mission as needed.
That said, i'm not sure she will have time to change also `LSST` and `Generic` case but as is not part of the release, that could be done anytime, not necessarily now unless she managed to do it or somebody have time to do it by end of this month. 
_Bottom line_: The code she is adding, to support the changes, shouldn't affect the other mission specific cases as they are currently but it has to support the changes for WISE and Other at very least and for sure the near **future cases such as PTF**. 

If you need to discuss further, please come to me.